definitions/v1/mircrew.yml

```diff
@@ -0,0 +1,175 @@
+    - {id: 3, cat: Other, desc: "  Comunicazioni, Annunci & Guide"}
+    - {id: 11, cat: Other, desc: "CAG Progetti & Collezioni"}
+    - {id: 12, cat: Other, desc: "CAG Pir8 Channel"}
+    - {id: 13, cat: Other, desc: "CAG Progetti"}
+    - {id: 61, cat: Other, desc: "CAG Video"}
+    - {id: 62, cat: Other, desc: "CAG Animazione"}
+    - {id: 63, cat: Other, desc: "CAG Libreria"}
+    - {id: 64, cat: Other, desc: "CAG Musica"}
+    - {id: 14, cat: Other, desc: "CAG Collezioni"}
+    - {id: 50, cat: Other, desc: "CAG Proposte & Richieste"}
+    - {id: 60, cat: Other, desc: "CAG Abbandonati"}
+    - {id: 15, cat: Other, desc: "Area Grafica"}
+    - {id: 16, cat: Other, desc: "Vetrina MIRCrew"}
+    - {id: 17, cat: Other, desc: "Vetrina Utenti"}
+    - {id: 19, cat: Other, desc: "Area Staff"}
+    - {id: 23, cat: Other, desc: "Archivio"}
+    - {id: 25, cat: Movies, desc: "Video Releases"}
+    - {id: 26, cat: Movies, desc: "Releases Film"}
+    - {id: 28, cat: TV, desc: "Releases Serie TV"}
+    - {id: 51, cat: TV, desc: "Releases TV Stagioni in corso"}
+    - {id: 52, cat: TV, desc: "Releases TV Stagioni complete"}
+    - {id: 29, cat: TV/Documentary, desc: "Documentari"}
+    - {id: 30, cat: TV, desc: "TV Show"}
+    - {id: 31, cat: TV, desc: "Teatro"}
+    - {id: 32, cat: Movies, desc: "Richieste Video"}
+    - {id: 53, cat: Movies, desc: "Richieste Film"}
+    - {id: 54, cat: TV, desc: "Richieste Serie TV"}
+    - {id: 55, cat: Movies, desc: "Richieste Video diverse"}
+    - {id: 33, cat: TV/Anime, desc: "Animazione Releases"}
+    - {id: 34, cat: TV/Anime, desc: "Anime - Movies"}
+    - {id: 35, cat: TV/Anime, desc: "Anime - Serie"}
+    - {id: 36, cat: TV/Anime, desc: "Cartoon - Movies"}
+    - {id: 37, cat: TV/Anime, desc: "Cartoon - Serie"}
+    - {id: 38, cat: TV/Anime, desc: "Richieste Animazione"}
+    - {id: 39, cat: Books, desc: "Libreria Releases"}
+    - {id: 40, cat: Books/EBook, desc: "E-Books"}
+    - {id: 41, cat: Audio/Audiobook, desc: "A-Books"}
+    - {id: 42, cat: Books/Comics, desc: "Comics"}
+    - {id: 43, cat: Books/Mags, desc: "Edicola"}
+    - {id: 44, cat: Books, desc: "Richieste Libreria"}
+    - {id: 45, cat: Audio, desc: "Music Releases"}
+    - {id: 46, cat: Audio, desc: "Musica - Audio"}
+    - {id: 47, cat: Audio/Video, desc: "Musica - Video"}
+    - {id: 48, cat: Audio, desc: "Richieste Musica"}
+
+  modes:
+    search: [q]
+    tv-search: [q, season, ep]
+    movie-search: [q]
+    music-search: [q]
+    book-search: [q]
+
+settings:
+  - name: username
+    type: text
+    label: Username
+  - name: password
+    type: password
+    label: Password
+
+login:
+  path: ucp.php?mode=login
+  method: form
+  form: form#login
+  inputs:
+    username: "{{ .Config.username }}"
+    password: "{{ .Config.password }}"
+    autologin: on
+    viewonline: on
+  error:
+    - selector: div.error
+  test:
+    path: index.php
+    selector: a[href^="./ucp.php?mode=logout&sid="]
+
+download:
+  selectors:
```

Comment:
Download / Selectors is not valid in v1, this needs reworked and this version added to v2

Reply:
mircrew moved to v2;  will not be backported to v1 / will not be supported until https://github.com/Prowlarr/Prowlarr/pull/310 lands; similar to other indexers jackett has added that use download selectors.

Reply:
You COULD modify to work with v1 if you loved the users :P

Reply:
We have not done it for any other indexer; I see no reason to start now.